dataStandard_versioned_docs/version-6/model-reference/performance-evaluation-domain/entities-references-and-descriptors.md

```diff
@@ -0,0 +1,54 @@
+
+## Extended References
+
+| Name | Description |
+| --- | --- |
+| EducationOrganization | This entity represents any public or private institution, organization, or agency that provides instructional or support services to students or staff at any level. |
+| Person | The person whose performance is being evaluated. |
+| Section | The section associated with a classroom evaluation. |
+
+## Path Domain Descriptors
+
+| Entity | Name | Description | Usage Classification | EDFacts Mapping | Commonly Used | Commonly State-Defined |
+| --- | --- | --- | --- | --- | --- | --- |
+| PerformanceEvaluation | AcademicSubject | The description of the content or subject area of the performance evaluation. | Local | Yes | Yes |     |
+| PerformanceEvaluationRating | CoteachingStyleObserved | A type of co-teaching observed as part of the performance evaluation. | Standard |     |     |     |
+| EvaluationElementRating | EvaluationElementRatingLevel | The rating level achieved based upon the rating or score. | Flexibile |     |     |     |
+| PerformanceEvaluation | EvaluationPeriod | The period for the evaluation. | Orthodox |     |     |     |
+| EvaluationRating | EvaluationRatingLevel | The rating level achieved based upon the rating or score. | Flexible |     |     |     |
+| EvaluationRating | EvaluationRatingStatus | The status of the performance evaluation. | Standard |     |     |     |
+| Evaluation <br /> EvaluationElement <br /> EvaluationObjective | EvaluationType | The type of the evaluation or evaluation objective. | Standard |     |     |     |
+| Goal | GoalType | The type of the goal. |     |     |     |     |
+| PerformanceEvaluation | GradeLevel | The grade levels involved with the performance evaluation. | Orthodox | Yes | Yes |     |
+| EvaluationObjectiveRating | ObjectiveRatingLevel | The rating level achieved based upon the rating or score. | Flexible |     |     |     |
+| PerformanceEvaluation | PerformanceEvaluationType | The type of performance evaluation conducted. | Orthodox |     |     |     |
+| PerformanceEvaluationRating | PerformanceEvaluationRatingLevel | The rating level achieved based upon the rating or score. | Flexible |     |     |     |
+| QuantitativeMeasure | QuantitativeMeasureDatatype | The datatype of the result. The results can be expressed as a number, percentile, range, level, etc. |     |     |     |     |
+| QuantitativeMeasure | QuantitativeMeasureType | The type of the quantitative measure. |     |     |     |     |
+| EvaluationElementRating <br /> EvaluationObjectiveRating <br /> EvaluationRating <br /> PerformanceEvaluationRating <br /> | ResultDatatypeType | The datatype of the rating result. | Orthodox |     |     | Yes |
+| RubricDimension | RubricRatingLevel | The rating level achieved for the rubric dimension. | Flexible |     |     |     |
+| PerformanceEvaluation | Term | The term for the session during the school year. | Flexible |     |     |     |
```

Comment:
Should this page have the "tip" annotation at the bottom, with a link to the page describing the usage classifications? Please review the other new entity pages as well. I did not look closely at them.